app/Console/Commands/Migrations/CreateMissingAuth0Clients.php

```diff
@@ -84,7 +84,7 @@ private function createMissingAuth0ClientsForIntegration(Integration $integratio
      */
     private function getMigratedIntegrations(): Collection
     {
-        $migratedIntegrationModels = IntegrationModel::query()->whereNotNull('migrated_at')->get();
+        $migratedIntegrationModels = IntegrationModel::query()->get();
```

Comment:
Can we query here specifically to only get the integrations that do not have three auth0 clients? Is that not possible with Eloquent?